ansys/mapdl/core/mapdl.py

```diff
@@ -2164,10 +2164,89 @@ def ignore_errors(self):
     def ignore_errors(self, value):
         self._ignore_errors = bool(value)
 
-    def load_table(self, name, array, var1="", var2="", var3=""):
+    def load_array(self, name, array):
+        """Load an array from Python to MAPDL.
+
+        Uses VREAD to transfer the array.
+        The format of the numbers used in the intermediate file is F24.18.
+
+        Parameters
+        ----------
+        name : str
+            An alphanumeric name used to identify this table.  Name
+            may be up to 32 characters, beginning with a letter and
+            containing only letters, numbers, and underscores.
+            Examples: ``"ABC" "A3X" "TOP_END"``.
+
+        array : np.ndarray or List
+            List as a table or ``numpy`` array.
+
+        Examples
+        --------
+        >>> my_conv = np.array([[0, 0.001],
+                                [120, 0.001],
+                                [130, 0.005],
+                                [700, 0.005],
+                                [710, 0.002],
+                                [1000, 0.002]])
+        >>> mapdl.load_array('MY_ARRAY', my_conv)
+        >>> mapdl.parameters['MY_ARRAY']
+        array([[0.0e+00, 1.0e-03],
+                [1.2e+02, 1.0e-03],
+                [1.3e+02, 5.0e-03],
+                [7.0e+02, 5.0e-03],
+                [7.1e+02, 2.0e-03],
+                [1.0e+03, 2.0e-03]])
+        """
+        if not isinstance(array, np.ndarray):
+            raise ValueError("The array should be a Numpy array")
```

Comment:
Use `TypeError` rather than `ValueError`
```suggestion
            raise TypeError(f"The ``array`` parameter should be a Numpy array, not {type(array)}")
```